src/main/java/org/apache/bcel/classfile/JavaClass.java

```diff
@@ -906,4 +910,22 @@ public String toString() {
         }
         return buf.toString();
     }
+
+    public boolean isRecord() {
```

Comment:
Javadoc missing, the next version will be 6.9.0 since this is a PR for a new feature.